### PR TITLE
Translate app to french

### PR DESCRIPTION
**Pull request by blasius**
I propose in this commit a translation of the app in french. All translatable strings have been given their french translation. Thank you

### CODE REVIEW

**Comment by or-else** [comment]
Thanks for the translation! Could you please double check the strings against `devel` branch. Quite a few things have changed in the last release.

**Comment by blasius** [comment]
Sure Sir. It's done

**Comment by or-else** [comment]
Thanks a lot!

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+
+    <string name="app_name" translatable="false">Tinode</string>
+    <string name="app_name_full" translatable="false">Tinode Chat</string>
+    <string name="app_version">Version</string>
+    <string name="copyright" translatable="false">© 2014 – 2021 Tinode</string>
+    <string name="tinode" translatable="false">Tinode</string>
+
+    <string name="login_hint">Connexion</string>
+    <string name="password_hint">Mot de passe</string>
+    <string name="new_message_hint">Nouveau message</string>
```
Comment (liyuanmeng1) [inline app/src/main/res/values-fr/strings.xml:15]: text

File: app/src/main/res/values-fr/strings.xml [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+    <string name="send_message_button_hint">Envoyer message</string>
+    <string name="send_image_button_hint">Envoyer une image</string>
+    <string name="send_file_button_hint">Envoyer un fichier</string>
+
+    <string name="sign_in">Se connecter</string>
+    <string name="app_description" translatable="false">Tinode Chat</string>
+    <string name="action_signup">S\'inscrire</string>
+    <string name="sign_up">S\'inscrire</string>
+    <string name="login_new">Identifiant</string>
+    <string name="password_new">Mot de passe</string>
```
Comment (liyuanmeng1) [inline app/src/main/res/values-fr/strings.xml:25]: text

**Comment by or-else** [comment]
Last warning.

**Pull request merged**